src/engine-actions/close-channels.js

```diff
@@ -0,0 +1,70 @@
+const {
+  listChannels,
+  closeChannel
+} = require('../lnd-actions')
+/**
+ * Closes a channel given the channel point
+ *
+ * @param {Boolean} [options.outbound=true] outbound is true if checking outbound channels, false if inbound
+ * @returns {Promise<Array.<Object>>}
+ */
+async function closeChannels ({ force = false } = {}) {
+  const { channels = [] } = await listChannels({ client: this.client })
+  const activeChannels = channels.filter(c => c.active)
+
+  const closedChannelResponses = await Promise.all(activeChannels).map(channel => close(channel, force, this.client, this.logger))
+
+  this.logger.info('Successfully closed channels', closedChannelResponses)
+  return closedChannelResponses
+}
+
+async function close (channel, client, force, logger) {
+  logger.info('FORCE', force)
```

Comment:
@martineehrlich did you still want this log? it looks like its a dev log

Reply:
I'm using it for now haha, but I'll remove it when im done testing